indico/web/templates/_session_bar.html

```diff
@@ -6,175 +6,195 @@
-            {% elif mode == 'network' %}
-                {% trans count=networks|length, networks=networks|join('/') %}
-                    The information on this page is restricted for display on the {{ networks }} network only.
-                {% pluralize %}
-                    The information on this page is restricted for display on the {{ networks }} networks only.
-                {% endtrans %}
-            {% elif mode == 'restricted' %}
-                {% trans %}
-                    The information on this page is restricted for display to named individuals or specific groups.
-                {% endtrans %}
-            {% endif %}
-        </div>
-        {% set disclaimer = disclaimers[mode]|sanitize_html|trim %}
-        {% if disclaimer %}
+    <ind-with-toggletip>
+        <button id="protection-details-link"
+           data-protection-mode="{{mode}}"
+           data-trigger
+           class="popup-trigger">
+            <span class="protection-mode-label">{% trans %}Page access restriction:{% endtrans %}</span>
+                {% if mode == 'public' %}
+                    {% trans %}Public{% endtrans %}
+                {% elif mode == 'network' %}
+                    {% trans count=networks|length, networks=networks|join('/') %}
+                        {{ networks }} network only
+                    {% pluralize %}
+                        {{ networks }} networks only
+                    {% endtrans %}
+                {% elif mode == 'restricted' %}
+                    {% trans %}Restricted{% endtrans %}
+                {% endif %}
+            <span class="protection-mode-label">{% trans %}(activate for details){% endtrans %}</span>
+        </button>
+
+        <div data-tip-content id="protection-details" aria-live="polite">
             <div class="protectionWidgetSection">
-                {{ disclaimer }}
+                {% if mode == 'public' %}
+                    {% trans %}The information on this page is visible to anyone{% endtrans %}
+                {% elif mode == 'network' %}
+                    {% trans count=networks|length, networks=networks|join('/') %}
+                        The information on this page is restricted for display on the {{ networks }} network only.
+                    {% pluralize %}
+                        The information on this page is restricted for display on the {{ networks }} networks only.
+                    {% endtrans %}
+                {% elif mode == 'restricted' %}
+                    {% trans %}
+                        The information on this page is restricted for display to named individuals or specific groups.
+                    {% endtrans %}
+                {% endif %}
             </div>
-        {% endif %}
-    </div>
-
-    <button id="protection-details-link"
-       class="arrow i-button protection-{{ mode }} {{ 'icon-shield' if mode != 'public' else 'icon-earth' }}">
```

Comment:
Did you remove the earth icon for public content on purpose?

Reply:
Yes and no. I was going to re-add it via CSS then it slipped my mind.